configtx/config.go

```diff
@@ -274,7 +287,7 @@ func newSystemChannelGroup(channelConfig Channel) (*cb.ConfigGroup, error) {
 
 	err = setPolicies(channelGroup, channelConfig.Policies, AdminsPolicyKey)
 	if err != nil {
-		return nil, fmt.Errorf("failed to set system channel policies: %v", err)
+		return nil, fmt.Errorf("failed to add system channel policies: %v", err)
```

Comment:
looks like the `s/set/add/` was dropped.

Reply:
Good catch. Done.